skatepark/block-templates/single.html

```diff
@@ -9,7 +9,7 @@
 
 <!-- wp:group {"align":"full"} -->
 <div class="wp-block-group alignfull">
-<!-- wp:post-featured-image {"align":"full"} /--></div>
+<!-- wp:post-featured-image {"align":"full", "style":{"spacing":{"margin":{"bottom":"calc(2*var(--wp--style--block-gap) + 3*var(--wp--custom--gap--horizontal))"}}}} /--></div>
```

Comment:
I don't think we should be using `var()` or `calc()` in block markup; It doesn't work in the FSE when you load that block template.  It makes the block markup more verbose, but I'm of the opinion that we should be verbose there, even if it means more manual changes when things like "block gap" needs to change.

Reply:
About this, looks like there's a proposal so that those values are [not empty](https://github.com/WordPress/gutenberg/issues/35558) on the block sidebar

Reply:
@MaggieCabrera Perhaps this PR could be moved to `Approved but not mergable`?

Reply:
no, I think we can merge this anyway, since only the UI is affected by the future change and not the actual looks of the site